services/api-server/src/simcore_service_api_server/utils/client_base.py

```diff
@@ -52,9 +52,14 @@ async def request(self, method: str, url: URLTypes, **kwargs):
         _logger.info("Capturing %s ... [might be slow]", capture_name)
         try:
             capture_json = get_captured_as_json(name=capture_name, response=response)
-            _capture_logger.info("%s", capture_json)
-        except ValidationError:
-            _capture_logger.exception("Failed capturing %s", capture_name)
+            _capture_logger.info("%s,", capture_json)
+        except Exception:
```

Comment:
Not great using `Exception` but i guess it is acceptable in this case.
Probably pylint will fail here. YOu need todisable it here

Reply:
I use a more specific exception now